test/document.test.js

```diff
@@ -9834,4 +9834,35 @@ describe('document', function() {
       assert.equal(fromDb.arr[0].abc, 'ghi');
     });
   });
+
+  it('supports getting a list of populated docs (gh-9702)', function() {
+    const Child = db.model('Child', Schema({ name: String }));
+    const Parent = db.model('Parent', {
+    children: [{ type: ObjectId, ref: 'Child' }],
+    child: {type: ObjectId, ref: 'Child'}
+    });
+    
+    return co(function*() {
+    const c = yield Child.create({ name: 'test' });
```

Comment:
There's some issues with indentation here. Run `npm run lint -- --fix` and that should help fix these errors.